subsys/usb/device/class/cdc_acm.c

```diff
@@ -537,9 +536,7 @@ static int cdc_acm_fifo_fill(const struct device *dev,
 	lock = irq_lock();
 	wrote = ring_buf_put(dev_data->tx_ringbuf, tx_data, len);
 	irq_unlock(lock);
-	if (wrote < len) {
-		LOG_WRN("Ring buffer full, drop %zd bytes", len - wrote);
-	}
+	LOG_DBG("Wrote %zd of %zd bytes to TX ringbuffer", wrote, len);
```

Comment:
len is just `int`, but `size_t` is unsigned.

LOG_DBG("Wrote %zu of %d bytes to TX ringbuffer", wrote, len);